### PR TITLE
Feature/ce03

### PR DESCRIPTION
**Pull request by CameoDaManeo**
Added skeleton code for future autocategorisation

Cleaned up jdoc comments in csvParsing.ts

### CODE REVIEW

File: src/pages/Accounts/ImportTransaction.tsx [hunk 1]
```diff
@@ -6,26 +6,21 @@ import { Transaction } from "../../types/transaction";
...
+
+  const processCSV = async (file: File) => {
     if (file) {
-      // TODO: properly format the csv parser and its returns
-      // TODO: Link importing to an account
+      // Generate a list of transactions linked to an account, along with import data and dupe indexes
       const importWithDupeIndexes = await generateImportFromFile(
         file,
-        "test"
+        accountId,
```
Comment (Maverick-Run) [inline src/pages/Accounts/ImportTransaction.tsx:20]: image

Comment (Maverick-Run) [reply src/pages/Accounts/ImportTransaction.tsx:20]: Need to fix this type error before the pull can be confirmed

**Review by Maverick-Run — CHANGES_REQUESTED** [review]
Need to fix the account ID type error in the import transaction before pull can be commited

File: src/pages/Accounts/ImportTransaction.tsx (hunk 1 above)
Comment (CameoDaManeo) [reply src/pages/Accounts/ImportTransaction.tsx:20]: Done :)

Hope this fix is OK

**Review by Maverick-Run — APPROVED** [review]
loks good

**Pull request merged**